spec/std/isa/csr/scountinhibit.yaml

```
@@ -0,0 +1,1178 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: scountinhibit
+long_name: Supervisor Counter-Inhibit Register
+address: 0x120
+writable: true
+priv_mode: S
+description: |
+  For counters delegated to S-mode, the associated `mcountinhibit` bits can be accessed
+  via `scountinhibit`. For counters not delegated to S-mode, the associated bits in
+  `scountinhibit` are read-only zero.
+length: SXLEN
```

Since this is a "masked alias of `mcountinhibit`", we need some adjustments here, including:
```suggestion
length: 32
```

Further, all of the fields need to be defined as `alias` of the corresponding field of `mcountinhibit`.

---

Done!